### PR TITLE
[Ready for Review] Bahasa Indonesia - Tips for Getting Started

### PR DESCRIPTION
**Pull request by frirasyidi**
Introduce Bahasa Indonesia translation for the page:
- index
- designing
- developing
- writing

### CODE REVIEW

**Comment by frirasyidi** [comment]
@remibetin on designing page, I found this issue: https://github.com/w3c/wai-quick-start/issues/394, fix is already included in Bahasa Indonesia translation but not in others.

**Comment by remibetin** [comment]
> @remibetin on designing page, I found this issue: https://github.com/w3c/wai-quick-start/issues/394, fix is already included in Bahasa Indonesia translation but not in others.

Thanks! Fix is now live in English & French versions 🎉

**Review by remibetin — CHANGES_REQUESTED** [review]
- Index page: reviewed: looks good to me ✅
- Designing: reviewed, see my few comments 💬 
- Writing: review in progress
- Developing: review in progress

File: _tips/writing.id.md [hunk 1]
```diff
@@ -0,0 +1,470 @@
...
+{:/}
+
+<div class="heading-structure two-column">
+  <figure>
+    <figcaption>{% include_cached icon.html name="ex-circle" label="Salah" %} Kekurangan judul</figcaption>
+    <div>
+      {% include_cached img.html src="headings-poor.png" alt="Contoh tidak ada judul, lihat di bawah untuk contoh mendetail" %}
+    </div>
+{% include_cached excol.html type="start" id="no-headings" %}
+Lihat contoh *inline*
```
Comment (remibetin) [inline _tips/writing.id.md:159]: @frirasyidi If "inline" is an anglicism, would you wrap it with `<span lang"en"></span>`?

File: _tips/writing.id.md [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+<cite>Diadopsi dari <a href="http://accessibility.psu.edu/headings/">"Judul dan Subjudul" di Penn State</a></cite>
+{% include_cached excol.html type="end" %}
+  </figure>
+  <figure>
+    <figcaption>{% include_cached icon.html name="check-circle" label="Oke" %} Menggunakan judul dan subjudul</figcaption>
+    <div>
+      {% include_cached img.html src="headings-good.png" alt="Ilustrasi struktur judul yang bagi, lihat di bawah untuk contoh mendetail"%}
+    </div>
+{% include_cached excol.html type="start" id="headings-good" %}
+Lihat comtoh *inline*
```
Comment (remibetin) [inline _tips/writing.id.md:184]: @frirasyidi If "inline" is an anglicism, would you wrap it with `<span lang"en"></span>`?

File: _tips/writing.id.md [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
+* **User Story**
+  * [Menjelaskan nilai dari alternatif teks bagi pengguna penyandang disabilitas netra](/people-use-web/user-stories/#accountant)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Buat transkripsi dan takarir untuk multimedia
+
+Untuk konten audio, seperti podcast, berikan transkripsinya. Untuk konten audiovisual, seperti video pelatihan, sediakan pula takarirnya. Cantumkan pada transkripsi dan takarir informasi terkait ucapan lisan dan suara yang penting diketahui untuk memahami konten, misalnya, 'pintu berderit'. Untuk transkripsi video, sertakan juga deskripsi konten visual yang penting, misalnya 'Athan meninggalkan ruangan'.
```
Comment (remibetin) [inline _tips/writing.id.md:323]: Original content: "For audio-only content, such as a podcast, provide a transcript."

Translation: "Untuk konten audio, seperti podcast, berikan transkripsinya"

@frirasyidi Could you specify "audio-only" (not only "audio") in your translation?

File: _tips/writing.id.md [hunk 4]
```diff
@@ -0,0 +1,470 @@
...
+
+{::nomarkdown}
+{% include box.html type="start" title="Contoh: Pesan eror menunjukkan apa masalahnya dan, sebisa mungkin, bagaimana cara memperbaikinya" class="example" %}
+{:/}
+
+<div class="errors">
+  <figure>
+    <div>
+      <ol class="fa-ul error-list">
+        <li id="error_email">{% include_cached icon.html name="warning" label="Eror" %} <a href="javascript:return false">Nama pengguna 'superbear' sudah ada yang menggunakan.</a></li>
```
Comment (remibetin) [inline _tips/writing.id.md:375]: @frirasyidi If you think best to keep the English "superbear", please wrap it in `<span lang="en"></span>`

**Review by remibetin — CHANGES_REQUESTED** [review]

File: _tips/developing.id.md [hunk 5]
```diff
@@ -0,0 +1,638 @@
...
+* **Tutorial**
+  * [Melabeli Kontrol](/tutorials/forms/labels/)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Sediakan alternatif teks untuk gambar
+
+Pastikan alternatif teks untuk gambar ditambahkan ke semua gambar yang bersifat informatif dan fungsional. Gunakan teks alternatif kosong, `alt=""` untuk gambar dekoratif, atau sertakan gambar tersebut pada CSS. Alternatif teks biasanya disediakan oleh mereka yang bertanggung jawab untuk menulis konten.
```
Comment (remibetin) [inline _tips/developing.id.md:115]: Original content: "Ensure that alternative text for images is added to all informational and functional images."

Translation: "Pastikan alternatif teks untuk gambar ditambahkan ke semua gambar yang bersifat informatif dan fungsional."

@frirasyidi I just want to make sure that the translation does not imply that the images have to be functional AND informative for the alternative text to be necessary. Said differently, we want alt text added to all informative images, and all functional images.
Would you say the current translation could be misleading, or not?

File: _tips/developing.id.md [hunk 6]
```diff
@@ -0,0 +1,638 @@
...
+* **Kisah Pengguna**
+  * [Menjelaskan nilai dari alternatif teks bagi pengguna penyandang disabilitas netra](/people-use-web/user-stories/#accountant)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Identifikasi bahasa halaman dan perubahan bahasa
+
+Tunjukkan bahasa utama setiap halaman dengan menggunakan atribut `lang` pada tag `html`, misalnya `<html lang="en">`. Gunakan atribut `lang` pada elemen jika bahasa elemen tersebut berbeda dari elemen lainnya pada halaman.
```
Comment (remibetin) [inline _tips/developing.id.md:134]:
```suggestion
Tunjukkan bahasa utama setiap halaman dengan menggunakan atribut `lang` pada tag `html`, misalnya `<html lang="id">`. Gunakan atribut `lang` pada elemen jika bahasa elemen tersebut berbeda dari elemen lainnya pada halaman.
```

I think it is appropriate for this translation to use this value.

File: _tips/developing.id.md [hunk 7]
```diff
@@ -0,0 +1,638 @@
...
+* **Kisah Pengguna**
+  * [Menjelaskan bagaimana alternatif tampilan pada halaman yang dizum dapat membantu](/people-use-web/user-stories/#retiree)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Sediakan makna untuk elemen interaktif yang nonstandar
+
+Gunakan <abbr title="Aplikasi Internet yang Kaya dan Aksesibel">WAI-ARIA</abbr> untuk memberikan informasi tentang fungsi dan status *widget* yang dirancang khusus, seperti akordeon dan tombol yang dirancang khusus. Misalnya, `role="navigation"` dan `aria-expanded="true"`. Kode tambahan diperlukan untuk mengimplementasikan perilaku *widget* tersebut, seperti memperluas dan menyusutkan konten, atau bagaimana *widget* merespons aksi dari kibor.
```
Comment (remibetin) [inline _tips/developing.id.md:467]: @frirasyidi Please wrap "widget" in `<span lang="en"></span>`.

**Review by remibetin — CHANGES_REQUESTED** [review]

File: _tips/designing.id.md [hunk 8]
```diff
@@ -0,0 +1,673 @@
...
+    <figure class="hover">
+      <figcaption>{% include_cached icon.html name="check-circle" label="Oke" %} Gaya ketika disoroti tetikus</figcaption>
+      <div>
+        <p style="text-align: center; position:relative;"><a href="javascript:return false;" aria-label="Tautan diberi gaya dengan latar belakang yang halus untuk menyorot saat disoroti" style="color: #850BAC; background-color: rgba(253, 247, 15, 0.3)">navigasi dengan kibor</a><img src="../img/pointer.png" style="position: absolute; left: 50%; top:1em;" width="20" alt=""></p>
+      </div>
+    </figure>
+    <figure class="keyboard">
+      <figcaption>{% include_cached icon.html name="check-circle" label="OK" %} Gaya ketika difokuskan via kibor</figcaption>
+      <div>
+        <p style="text-align: center; position:relative;"><a href="javascript:return false;" aria-label="Tautan diberi gaya dengan versi gaya disoroti yang lebih kuat untuk membuatnya menonjol" style="color: #850BAC; background-color: #fdf70f">>navigasi dengan kibor</a></p>
```
[the remark this answers is not in the document]
Comment (frirasyidi) [reply _tips/designing.id.md:263]: Fixed on the next commit.

File: _tips/designing.id.md [hunk 9]
```diff
@@ -0,0 +1,673 @@
...
+* **Kisah Pengguna**
+  * [Bagaimana konsistensi dan navigasi dapat membantu seseorang dengan kesulitan kognitif](/people-use-web/user-stories/#supermarketassistant)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Pastikan elemen isian disertai label yang jelas
+
+Pastikan semua bidang memiliki label deskriptif yang bersebelahan dengan bidang tersebut. Untuk bahasa dari kiri ke kanan, label biasanya diposisikan di kiri atau di atas bidang, kecuali untuk kotak centang dan tombol radio yang biasanya diletakkan di kanan elemen. Hindari memberikan terlalu banyak ruang antara label dan bidangnya.
```
[the remark this answers is not in the document]
Comment (frirasyidi) [reply _tips/designing.id.md:314]: @remibetin Yep, it does. The "which" part refers to the label. I add the "element" at the end of the sentence to make it clearer that the "which" here refers to the label.

I can improve the translation to have less ambiguity: "Untuk bahasa yang ditulis dari kiri ke kanan, label biasanya diposisikan di sebelah kiri atau atas bidang, kecuali untuk kotak centang dan tombol radio, label biasanya diletakkan di sebelah kanan."

I'll made the change when you've approved it.

File: _tips/writing.id.md (hunk 1 above)
Comment (frirasyidi) [reply _tips/writing.id.md:159]: @remibetin true! I missed that.
Fixed on the next commit.

File: _tips/writing.id.md (hunk 2 above)
Comment (frirasyidi) [reply _tips/writing.id.md:184]: @remibetin Fixed on the next commit.

File: _tips/writing.id.md (hunk 3 above)
Comment (frirasyidi) [reply _tips/writing.id.md:323]: @remibetin can and done in the next commit.

File: _tips/writing.id.md (hunk 4 above)
Comment (frirasyidi) [reply _tips/writing.id.md:375]: @remibetin 'superbear' is a name so I should not wrap it in <span lang="en"></span>, no?

File: _tips/developing.id.md (hunk 5 above)
Comment (frirasyidi) [reply _tips/developing.id.md:115]: @remibetin got it! I changed it to "Pastikan alternatif teks untuk gambar ditambahkan ke semua gambar yang bersifat informatif mau pun fungsional." in the next commit.

"mau pun" is similar to and/or.

File: _tips/developing.id.md (hunk 6 above)
Comment (frirasyidi) [reply _tips/developing.id.md:134]: @remibetin okay, added this to the next commit.

File: _tips/developing.id.md (hunk 7 above)
Comment (frirasyidi) [reply _tips/developing.id.md:467]: @remibetin done in the next commit.

File: _tips/designing.id.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (remibetin) [reply _tips/designing.id.md:314]: @frirasyidi 
 
I will leave it to you to judge which translation seems clearer and more natural. I just wanted to make sure the original meaning is preserved. Your confirmation was all I needed.

[the remark this answers is not in the document]
Comment (frirasyidi) [reply _tips/designing.id.md:314]: I've decided to change the translation to ensure clarity.

**Comment by remibetin** [comment]
QA done: ✅

**Comment by frirasyidi** [comment]
> @frirasyidi Thanks for the edits!
> 
> I have resolved all comments but [#393 (comment)](https://github.com/w3c/wai-quick-start/pull/393#discussion_r1521005465)
> 
> I will let you decide whether you want to make a final change or leave the translation as is. When you are done, would you just update the `last_updated` dates?

I have updated accordingly.

**Pull request merged**